assets/javascripts/hours_tracker.js

```diff
@@ -60,16 +60,9 @@ export default class HoursTracker {
     // TODO: rewrite and implement cleaner
     tickTimer(timerObject) {
         const updateTime = () => {
-            timerObject.seconds += 1;
-            if (timerObject.seconds >= 60) {
-                timerObject.seconds = 0;
-                timerObject.minutes += 1;
-                if (timerObject.minutes >= 60) {
-                    timerObject.minutes = 0;
-                    timerObject.hours += 1;
-                }
-            }
-            this.displayTimer(timerObject);
+            const [difference, tick] = this.timeDifference();
```

Comment:
This now calculates the time difference between start of the timer and current time... rather than counting every second.. which did not work during browser "sleep"